File: app/src/main/java/org/michaelbel/moviemade/rest/model/Language.java
package org.michaelbel.moviemade.rest.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

import io.realm.RealmObject;

public class Language extends RealmObject implements Serializable {

    @SerializedName("iso_639_1")
    public String language;

    @SerializedName("name")
    public String name;
}

File: app/src/main/java/org/michaelbel/moviemade/data/dao/Language.kt
package org.michaelbel.moviemade.data.dao

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Language(
    @SerializedName("iso_639_1") val language: String,
    @SerializedName("name") val name: String
) : Serializable